core/start-blueos-core

```diff
@@ -71,7 +71,7 @@ fi
 PRIORITY_SERVICES=(
     'autopilot',250,"nice --19 $SERVICES_PATH/ardupilot_manager/main.py"
     'cable_guy',250,"$SERVICES_PATH/cable_guy/main.py"
-    'video',250,"nice --19 mavlink-camera-manager --default-settings BlueROVUDP --mavlink tcpout:127.0.0.1:5777 --gst-feature-rank omxh264enc=0,v4l2h264enc=250,x264enc=260 --log-path /var/logs/blueos/services/mavlink-camera-manager --verbose"
+    'video',1000,"nice --19 mavlink-camera-manager --default-settings BlueROVUDP --mavlink tcpout:127.0.0.1:5777 --gst-feature-rank omxh264enc=0,v4l2h264enc=250,x264enc=260 --log-path /var/logs/blueos/services/mavlink-camera-manager --verbose"
```

Comment:
![meme-memes](https://github.com/bluerobotics/BlueOS/assets/1215497/39eac98e-46d3-414d-bbaa-11a6016fe9a8)

Reply:
dont look at me :eyes:

Reply:
@joaoantoniocardoso is that expected with the leak ?

Reply:
in my tests mcm started at about 350mb, going then with light cockpit usage it stayed under 500mb. but idk how it behaves with multiple cameras and/or 4k streams